examples/Vwell_2D.jl

```diff
@@ -1,11 +1,11 @@
 ## A simple 2D example for fluid-flow simulation
 
 using DrWatson
-@quickactivate "JutulDarcyRules-example"
+@quickactivate "JutulDarcyRules"
```

Comment:
Don't remember if julia allows to puth using/... in a try/catch statement so that if people don't use DrWatson it stills runs with standard package install